src/main/java/com/group3/backend/ui/model/request/MedicationDetailsModel.java

```diff
@@ -0,0 +1,80 @@
+package com.group3.backend.ui.model.request;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity(name="medication")
+public class MedicationDetailsModel {
+	
+
+	@Id
+	@GeneratedValue (strategy = GenerationType.AUTO)
+	private Integer id;
+	
+	private String description;
+	private String medicationName;
+	private String medicationClass;
+	private String dosage;
+	private String pharmacyName;
+	private Integer prescriptionCount;
```

Comment:
Don't need any of this stuff. Take a look at the ER diagram - if we want to display the pharmacy it's related to, we need to access the MedicationForResident class and get it that way